### DISCUSSION TITLE
Include AALSCharacter in c++

### DISCUSSION THREAD

**Discussion by dungeon2567** [Sixze/ALS-Refactored#359] [Q&A]
How do i create a custom ACharacter in c++? Is it possible?

![image](https://github.com/Sixze/ALS-Refactored/assets/31074903/203cf823-9e2b-4ac4-81ab-9346f6c7a739)

**Comment by JoeGitcpp** [comment]
I think you should create a new class that inherits from ALSCharacter because the MovementComponent of ALSCharacter is custom. Alternatively, you can try replacing the character's movement component with ALSMovementComp
